README.md

```diff
@@ -391,6 +391,55 @@ property limits styles to highlighted languages.
 When converting a Prism CSS theme it's mostly just necessary to use classes as
 `types` and convert the declarations to object-style-syntax and put them on `style`.
 
+### SSR Support / Avoiding FOUC
+
+If your React app supports "light mode / dark mode" you need to do additional work to avoid a flash of unstyled content (FOUC). Generate the following script tag and inject it into your HTML so it runs _before_ your content loads.
+
+```js
+import { generateScriptForSSR } from 'prism-react-renderer'
+import duotoneDark from 'prism-react-renderer/themes/duotoneDark';
+import duotoneLight from 'prism-react-renderer/themes/duotoneLight';
+
+// A stringified function returning the `id` of the
+// theme you wish to render on the initial page load
+const getThemeIdFuncStr = `
+  () => (
+    window.localStorage.get('color-mode') === 'dark'

```

Comment:
Do we want to add a note about how this should match up with how the end-user implements _storing_ the selected theme ID? E.g., if they use `window.localStorage.get('color-mode')` to derive color theme ID, then they'll need to update that `localStorage` key whenever their theme choice changes?

I can just imagine someone reading this, implementing this exact code, and then wondering why they're still seeing FOUC.

Reply:
Yeah! That's a great call out. I'll add some more example code.

Reply:
Done.